# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieServiceParent.cpp
# Commit: 3949d9b400d4
# Full Hash: 3949d9b400d41a64102657d2858b09c41de0f142
# Author: Paul Zuehlcke <pbz@mozilla.com>
# Date: 2023-09-07 21:34:02
# Regressor Bug: 1839918
# File Overlap Count: 3
# Description:
#   Bug 1839918 - Pass BrowsingContext ID and third-party flag through CookieService. r=bvandersloot,cookie-reviewers,edgul
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D185303
# ==============================================================================

diff -r 707021ebfd23 -r 3949d9b400d4 netwerk/cookie/CookieServiceParent.cpp
--- a/netwerk/cookie/CookieServiceParent.cpp	Thu Sep 07 08:25:55 2023 +0000
+++ b/netwerk/cookie/CookieServiceParent.cpp	Thu Sep 07 08:25:55 2023 +0000
@@ -246,8 +246,8 @@
   // we don't send it back to the same content process.
   mProcessingCookie = true;
 
-  bool ok = mCookieService->SetCookiesFromIPC(aBaseDomain, aOriginAttributes,
-                                              aHost, aFromHttp, aCookies);
+  bool ok = mCookieService->SetCookiesFromIPC(
+      aBaseDomain, aOriginAttributes, aHost, aFromHttp, aCookies, 0, false);
   mProcessingCookie = false;
   return ok ? IPC_OK() : IPC_FAIL(this, "Invalid cookie received.");
 }
